configure.ac

```diff
@@ -90,6 +90,20 @@ AC_ARG_WITH([tctidefaultconfig],
                                 ["The default TCTIs configuration string"])],
             [])
 
+AC_ARG_ENABLE([tcti-device],
+            [AS_HELP_STRING([--enable-tcti-device],
+                            [build the tcti-device module (default is yes)])],
+            [enable_tcti_device=$enableval],
+            [enable_tcti_device=yes])
+AM_CONDITIONAL([ENABLE_TCTI_DEVICE], [test "x$enable_tcti_device" != xno])
+
+AC_ARG_ENABLE([tcti-mssim],
+            [AS_HELP_STRING([--enable-tcti-mssim],
+                            [build the tcti-mssim module (default is yes)])],
+            [enable_tcti_mssim=$enableval],
+            [enable_tcti_mssim=yes])
+AM_CONDITIONAL([ENABLE_TCTI_MSSIM], [test "x$enable_tcti_mssim" != xno])
+
```

Comment:
We should check if at least one TCTI is enabled.

Reply:
I intentionally did not do so.

Reason being a case, where someone wants a SAPI in a firmware environment with their own (non-public) TCTI.
Thus, we want to compile without any of our "official" TCTIs.

Reply:
added it to the end, so it is easily spotted.